model-terms/model-terms-v1.1.md

```diff
@@ -0,0 +1,398 @@
+# **Program Addressability Framework (PAF) - Model Terms**
+
+**Version: 1.1**
+
+_Copyright and disclaimer statement_
+
+_Copyright © 2022 Shoosmiths LLP_
+
+_Everyone is permitted to copy or distribute verbatim copies of these terms, but changing these terms is not permitted. All distributed copies of these terms or any thing incorporating them must include this copyright and disclaimer statement._
+
+_These terms have been produced by the law firm Shoosmiths LLP. Nothing in these terms shall be construed as legal advice. Shoosmiths LLP and Prebid.org Inc disclaim all liability to any person using or relying on all or any part of these terms._
+
+## **1. DEFINITIONS**
+
+“**Binding Principles**” means the principles set out in clause 4.
+
+“**Breach Notification**” means a notice given in accordance with clause 8.2 that a Contracting Party has breached these Model Terms.
+
+“**Change in Law**” means any change to any Data Protection Laws or other applicable laws including any change in the interpretation of those laws by any court of competent jurisdiction or Supervisory Authority.
+
+“**Contracting Parties**” means the Parties to a Main Agreement and “**Contracting Party**” shall be construed accordingly.
+
+“**Controller**” means a person that determines the purposes and means of the Processing of Personal Data and “**Joint Controller**” means a Controller that does so jointly with one or more Controllers.
+
+“**Data Protection Laws**” means any legislation or other laws in any jurisdiction applicable to a Contracting Party that relate to the protection of personal data or privacy of natural persons including
+the use of cookies or similar technologies.
+
+“**Device**” means any web-enabled device or application accessing the PAF Network.
+
+“**Main Agreement**” means an agreement incorporating these Model Terms, alone or with other terms.
+
+“**Model Terms**” means these terms and conditions.
+
+“**Non-PAF Data**” means any data other than the PAF Data to which a Party may link a Pseudonymous Identifier, including the PAF Audit Log.
+
+“**PAF Audit Log**” means a log containing information received by a Party of each Transaction that has been Signed, which shall include at a minimum the PAF Data, timestamp, and transaction ID signed by the Sender.
+
+“**PAF Code**” means the software code relating to the Program available at https://github.com/prebid/addressability-framework/tree/main/mvp-spec.
+
+“**PAF Data**” means all Pseudonymous Identifiers and associated Preferences and PAF Metadata.
+
+“**PAF Ecosystem**” means each and every Party.
+
+**“PAF Metadata”** means the creator domain, timestamp, and cryptographic signature stored with the PAF Data or Raw PAF Data defined by the encoding scheme version at https://github.com/prebid/addressability-framework/tree/main/mvp-spec.
+
+“**PAF Network**” means each and every PAF Operator.
+
+"**PAF Network Name**” means the consumer facing trademark and associated branding defined at [insert URL].
```

Comment:
we should highlight those (e.g. yellow background like in the pdf). It's important because each actor will have to edit several [insert xxx] placeholders in this document.